demo/.storybook/main.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   stories: ['../../**/*.stories.json'],
   addons: [
-    '@storybook/addon-controls',
-    '@storybook/addon-a11y'
+    '@storybook/addon-a11y',
+    '@storybook/addon-controls'
```

Comment:
Not directly related to this PR, but thought it minor enough to slip in!
Switching the order of these per warning that appears on dev boot.